docs/cron-workflows.md

```diff
@@ -51,7 +51,8 @@ You can use `CronWorkflow.spec.workflowMetadata` to add `labels` and `annotation
 | `startingDeadlineSeconds`    | `0`                    | Seconds after [the last scheduled time](#crash-recovery) during which a missed `Workflow` will still be run. |
 | `successfulJobsHistoryLimit` | `3`                    | Number of successful `Workflows` to persist |
 | `failedJobsHistoryLimit`     | `1`                    | Number of failed `Workflows` to persist |
-| `stopStrategy`               | `nil`                  | v3.6 and after: defines if the CronWorkflow should stop scheduling based on a condition |
+| `stopStrategy`               | `nil`                  | v3.6 and after: defines if the CronWorkflow should stop scheduling based on an expression |
+| `stopStrategy.expression`    | None                   | v3.6 and after: an expression which if present must evaluate to false for the workflow to be created |
```

Comment:
I don't think it's necessary to explicitly explain both in the table as they are identical right now. You can instead link to the usage section below

```suggestion
| `stopStrategy`               | `nil`                  | v3.6 and after: defines if the CronWorkflow [should stop scheduling](#automatically-stopping-a-cronworkflow) based on an expression |
```

Reply:
I've left in the .expression part as I feel it should be possible to use this table to understand what to do, and `expression` is unguessable.

Reply:
That works, however the new description does not match what I wrote above, adding unnecessary and inconsistent verbosity, and missing a link. See follow-up thread